content/docs/2.0/scalers/azure-storage-blob.md

```diff
@@ -37,7 +39,11 @@ You can authenticate by using pod identity or connection string authentication.
 
 **Connection String Authentication:**
 
-- `connection` - Connection string for Azure Storage Account
+- `connection` - Connection string for Azure Storage Account.
+
+**PodIdentity Authentication**
```

Comment:
Nit, can you update the files to use this formatting, please?
```suggestion
**Pod Identity Authentication**
```

Reply:
I split 'PodIdentity' to 'Pod Identity'